Sources/FoundationEssentials/AttributedString/AttributeScope.swift

```diff
@@ -29,6 +29,7 @@ public enum AttributeScopes { }
 
 #if FOUNDATION_FRAMEWORK
 
+@_implementationOnly import Darwin
```

Comment:
We actually need just regular `import Darwin`. I believe this will cause the "inconsistently import" warning.